Watchman.AwsResources/Services/Sqs/QueueDataV2Source.cs

```diff
@@ -8,56 +8,16 @@ namespace Watchman.AwsResources.Services.Sqs
 {
     public class QueueDataV2Source : ResourceSourceBase<QueueDataV2>
     {
-        private readonly IAmazonCloudWatch _amazonCloudWatch;
+        private readonly QueueSource _queueSource;
```

Comment:
Each of these calls the AWS sdk, caches, and maps the response. So if we treat the new one as a wrapper around the old we will make half the calls.